pr_agent/git_providers/bitbucket_provider.py

```diff
@@ -10,12 +10,12 @@
 from ..config_loader import get_settings
 from ..log import get_logger
 from .git_provider import FilePatchInfo, GitProvider
+import ast
 
 
 class BitbucketProvider(GitProvider):
     def __init__(
-        self, pr_url: Optional[str] = None, incremental: Optional[bool] = False
-    ):
+        self, pr_url: Optional[str] = None, incremental: Optional[bool] = False):
         s = requests.Session()
         try:
             bearer = context.get("bitbucket_bearer_token", None)
```

Comment:
**Suggestion:** Remove the duplicate line that assigns the `bearer` variable. The line `bearer = context.get("bitbucket_bearer_token", None)` is repeated unnecessarily. [code cleanup, importance: 9]